torch/_dynamo/config.py

```diff
@@ -258,6 +258,20 @@ def is_fbcode():
     "skipfiles_inline_module_allowlist",
 }
 
+capture_autograd_function = True
+
+_autograd_backward_strict_mode_banned_ops = [
+    "stride",
+    "requires_grad",
+    "storage_offset",
+    "layout",
+    "is_cuda",
+    "is_quantized",
+    "is_meta",
+    "data",
+    "is_sparse",
```

Comment:
Could we do a regex and ban all `is_*` methods? The naming implies those are Tensor properties. Alternatively, we could add all of the ones in https://pytorch.org/docs/stable/tensors.html to the list, but that might not be as robust:
<img width="951" alt="image" src="https://github.com/pytorch/pytorch/assets/5652049/8d94a169-4510-40e1-a8bf-c5c134c33357">